$NetBSD: patch-image_h,v 1.1 2011/12/16 05:14:27 dholland Exp $

Yet another package that thinks it's a good idea to conditionalize on
someone else's include guards beyond its control...

--- image.h~	2005-03-20 20:15:32.000000000 +0000
+++ image.h
@@ -63,7 +63,7 @@ typedef struct {
     unsigned long sourceMask;
 } Image;
 
-#ifdef _XLIB_H_
+#if defined(_XLIB_H_) || defined(_X11_XLIB_H_)
 /* Used to transfer information to routines in iprocess.c */
 struct imageprocessinfo {
     int oilArea;
